packages/core/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@react-native-async-storage/core",
+  "version": "0.0.0",
+  "description": "Core API of Async Storage",
+  "main": "lib/commonjs/index",
+  "module": "lib/module/index",
+  "types": "lib/typescript/index.d.ts",
+  "react-native": "src/index",
+  "source": "src/index",
```

Comment:
We should also include an exports map. Metro started supporting it recently.

Also, why are we missing the extensions? Platform forking is not applied to entry point files regardless.

Reply:
I copied this from bob setup, so there was no file ext.
TIL about exports 😄  replaced it now